README.md

```diff
@@ -15,54 +17,56 @@
 The `dbt-cratedb2` package contains all the code enabling [dbt] to work with a
 [CrateDB] database.
 
-Because CrateDB is PostgreSQL-compatible, dbt-cratedb2
+CrateDB is PostgreSQL-compatible, that's why dbt-cratedb2
 heavily builds upon the canonical [dbt-postgres] adapter.
 For more information on using dbt with CrateDB,
-consult [the docs](https://docs.getdbt.com/docs/profile-cratedb).
+consult [the docs].
 
 ## About dbt
 
 dbt enables data analysts and engineers to transform their data using the same practices that software engineers use to build applications.
 
 dbt is the T in ELT. Organize, cleanse, denormalize, filter, rename, and pre-aggregate the raw data in your warehouse so that it's ready for analysis.
 
+## About CrateDB
+
+CrateDB is a distributed and scalable SQL database for storing and analyzing
+massive amounts of data in near real-time, even with complex queries.
+It is PostgreSQL-compatible, and based on Lucene.
+
 ## Getting started
 
-- [Install dbt](https://docs.getdbt.com/docs/installation)
-- Read the [introduction](https://docs.getdbt.com/docs/introduction/) and [viewpoint](https://docs.getdbt.com/docs/about/viewpoint/)
+- [Install dbt](https://docs.getdbt.com/docs/core/installation-overview)
+- Read the [introduction](https://docs.getdbt.com/docs/introduction/) and
+  [viewpoint](https://docs.getdbt.com/community/resources/viewpoint)
```

Comment:
Link to canonical page at the CrateDB Guide here.

- https://cratedb-guide--145.org.readthedocs.build/integrate/dbt/

Reply:
It's inside 0147433cd5ebc.